aksharamukha/transliterate.py

```diff
@@ -517,6 +518,43 @@ def convert_default(src, tgt, txt, nativize = True, post_options = [], pre_optio
 
     return convert(src, tgt, txt, nativize, pre_options, post_options)
 
+def convert_docx(src, tgt, file, nativize, pre_options, post_options):
+    new_file = io.BytesIO()      
+
+    with ZipFile(file, 'r') as old_archive:
+        with ZipFile(new_file, 'w') as new_archive:
+            for item in old_archive.filelist:
+                if item.filename.startswith('word/') and item.filename.endswith('.xml'):
+                    with old_archive.open(item, 'r') as xml_file:
+                        tree = etree.parse(xml_file)
+                        for element in tree.getroot().iter():
+                            if element.tag == "{http://schemas.openxmlformats.org/wordprocessingml/2006/main}t":
```

Comment:
consider refactoring this magic number to a constant with a telling name
```suggestion
                            if element.tag == MS_WORD_TEXT_NODE:
```